### PR TITLE
Fix css not loading on package pages

### PR DESCRIPTION
**Pull request by Meow-J**

### CODE REVIEW

File: _includes/themes/the-program/default.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	<title>{% if page.tabtitle %}{{ page.tabtitle }}{% else %}{{ page.title }}{% endif %}</title>
 
 	<meta name="author" content="{{ site.author.name }}">
-	<link href='{{ ASSET_PATH }}/css/style.css' rel="stylesheet" media="all">
+	<link href='{{ ASSET_PATH }}the-program/css/style.css' rel="stylesheet" media="all">
```
Comment (yorkie) [inline _includes/themes/the-program/default.html]: Should be `/the-program`?

Comment (Meow-J) [inline _includes/themes/the-program/default.html]: '{{ ASSET_PATH }}' actually includes a slash

Comment (yorkie) [inline _includes/themes/the-program/default.html]: - when `ASSERT_PATH` includes a slash, slash one is working.
- when `ASSERT_PATH` doesn't include a slash, slash one is still working, but the another one without slash will not work.

Comment (Meow-J) [inline _includes/themes/the-program/default.html]: alright, I'll just keep it.

**Comment by yorkie** [comment]
Thank you @Meow-J LGTM, cc @jwerle

**Pull request merged**